cgmanifest.json

```diff
@@ -27424,6 +27424,16 @@
           "downloadUrl": "https://github.com/facebook/zstd/releases/download/v1.5.0/zstd-1.5.0.tar.gz"
         }
       }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "usrsctp",
```

Comment:
Btw. this will be done for you, if you run this script:

```bash
toolkit/scripts/update_cgmanifest.py last cgmanifest.json SPECS/usrsctp/usrsctp.spec
```

Reply:
Acknowledged. Updated in the recent commit.